Congregated map changes

Ryan180602 opened the pull request:
:cl:
maptweak: Remove external bridge cameras. (B-Deck Bridge)
maptweak: Remap chemistry lab to be a bit bigger, while not taking up more space than normal in the medbay. (D-1 Med)
maptweak: Remove `NO IDLE` sign between cryo-lab, replace with bodyscan screen. (D-1 Med)
maptweak: Replace BC's crowbar with prybar. (D-1 Sec)
maptweak: Replace FT's cigarette pack spawn with a random cigarette pack spawn. (D-1 Sec)
maptweak: Add General Intercom to D1 Briefing Room. (D-1 Mid)
maptweak: Shift scrubber spawn a bit to not be inconvenient on ladders. (D-3 Behind SecCP)
maptweak: Remove glass panes between consoles. (D-3 Rec Room)
maptweak: Make PF office cameras belong to Exploration network. (D-4 PF Office)
maptweak: Replace Atmos Storage waste canisters with oxygen canisters. (D-5 Hangar)
/:cl:

Tiny map changes (minus the med one), that would be way too many PRs to open. But if asked, I can do that.

Chem-Lab remap has nothing added except for a ChemMaster. Everything else is (should be) just shifted around. 
ETC Remap only has the bodyscan console be an extra addition, and the `NO IDLE` sign as a removal.
Changed to (hopefully) look a bit bigger and different, and less congested, since we've had the current one for a while, now.

![image](https://github.com/Baystation12/Baystation12/assets/50071611/bd0a9bc8-513f-40d7-ac2d-93a20e0dfb4f)
vs
![image](https://github.com/Baystation12/Baystation12/assets/50071611/33a5575d-58f0-4fc4-b60b-4099e5cc08b5)



External cameras are useless (never used + no power, now).
BC's prybar is more in line with sec kit.
FT's cig pack can be randomised for flavour reasons.
General Intercom in briefing room, since usually every room has one.
Scrubber moved for convenience's sake.
Glass panes interfered with how the current joint sprites look.
PF Office should be under exploration network, not Deck 4.
No extra oxygen canisters, which ended up in Exploration/Science/Prospector having to steal prep-room cans. Can make them empty ones if needed but no proper place to fill them.

juxjux9930 commented:
Why two desks for the chemlab? you could do one and move a table to where the other chair is, and then tuck the disposals bin out of the middle of the room.

juxjux9930 commented:
Also- it’s been a while since i’ve looked at hangar atmos prep, but if the waste canisters are the yellow empty ones, they can just be relabeled to O2 canisters by anyone while they’re empty. Is filling them with O2 the problem?

Sbotkin commented:
> Also- it’s been a while since i’ve looked at hangar atmos prep, but if the waste canisters are the yellow empty ones, they can just be relabeled to O2 canisters by anyone while they’re empty. Is filling them with O2 the problem?

You cannot fill them anywhere but the atmospherics.

Ryan180602 commented:
> Why two desks for the chemlab? you could do one and move a table to where the other chair is, and then tuck the disposals bin out of the middle of the room.

Looks a bit better imo. It's entirely subjective, and from a practical sense, can be for two seperate lines of people. I can change it if it does not look good

> Also- it’s been a while since i’ve looked at hangar atmos prep, but if the waste canisters are the yellow empty ones, they can just be relabeled to O2 canisters by anyone while they’re empty. Is filling them with O2 the problem?

What sbotkin said. No place to fill them in other than in atmospherics. Since the connector for the airpump fills air-mix.

MuckerMayhem reviewed (dismissed):
Looks like you missed the external camera center to the bridge.

SierraKomodo reviewed (dismissed):
There's only one chemist slot. The chemistry lab doesn't need a second workstation.

Ryan180602 committed:
congregated map changes

remove external bridge cams, chemlab remap (only added an extra chemmaster), addition of bodyscan screen to ETC, change BC's crowbar to prybar, change FT's cig to random cig, add general intercom to briefing, remove windowpanes in rec room computers, change PF office cam to exploration, add two oxygen canisters instead of empty yellow ones, 
a


a


a


a


a


a

Ryan180602 commented:
fixed as above.
